address-space-controller/src/main/java/io/enmasse/controller/auth/CertManager.java

```diff
@@ -16,6 +18,9 @@
     Collection<CertComponent> listComponents(String uuid);
     boolean certExists(CertComponent component);
     Secret getCertSecret(String name);
+    default Secret getCertSecret(AddressSpace addressSpace) {
+        return getCertSecret(KubeUtil.getAddressSpaceExternalCaSecretName(addressSpace));
```

Comment:
I think this breaks the level of abstraction in CertManager.

I would prefer to not have this default method here and instead use KubeUtil.getAddressSpaceExternalCaSecretName(addressSpace) in EndpointController. 

Also because the address space CA is specific to the SelfSignedCertProvider.

Reply:
Done and rebased.